utils/copy-env-vars/.eslintrc.js

```diff
@@ -0,0 +1,58 @@
+const baseParserOptions = {
```

Comment:
Can we remove this file as root `.eslintrc.js` should be used in this instance rather than individual directories to have their own lint rules.